docs_src/query_param_models/tutorial001.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from fastapi import FastAPI, Query
+from pydantic import BaseModel, Field
+from typing_extensions import Literal
+
+app = FastAPI()
+
+
+class FilterParams(BaseModel):
+    limit: int = Field(100, gt=0, le=100)
+    offset: int = Field(0, ge=0)
+    order_by: Literal["created_at", "updated_at"] = "created_at"
```

Comment:
Should we probably add an alias for this tests?

```suggestion
    order_by: Literal["created_at", "updated_at"] = Field(alias="orderBy", default="created_at")
```

What is about `Header` and `Cookie`?

Reply:
Why do you think so? :thinking: 

Maybe you can suggest it in a PR on top after this. :nerd_face:

Reply:
After some issues between 0.112.4 and 0.114.1 with the alias I thought it could be a good idea.
I will make a suggestion.